src/ExpressionEvaluator/CSharp/Test/ResultProvider/FullNameTests.cs

```diff
@@ -15,6 +16,46 @@ namespace Microsoft.CodeAnalysis.CSharp.ExpressionEvaluator.UnitTests
     public class FullNameTests : CSharpResultProviderTestBase
     {
         [Fact]
+        public void Null()
+        {
+            IDkmClrFullNameProvider fullNameProvider = new CSharpFormatter();
+            var inspectionContext = CreateDkmInspectionContext();
+            Assert.Equal("null", fullNameProvider.GetClrExpressionForNull(inspectionContext));
+        }
+
+        [Fact]
+        public void This()
+        {
+            IDkmClrFullNameProvider fullNameProvider = new CSharpFormatter();
+            var inspectionContext = CreateDkmInspectionContext();
+            Assert.Equal("this", fullNameProvider.GetClrExpressionForThis(inspectionContext));
+        }
+
+        [Fact]
+        public void Cast()
```

Comment:
It would be nice to add a test to verify:
GetClrCastExpression(inspectionContext, "...", type, DkmClrCastExpressionOptions.ConditionalCast)
to get "(... as C)"
This format is used by the Null reference feature in the IDE.